### ISSUE TITLE
Deferred doc fix

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#10838]
|<img alt="bruce's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| bruce reported|
|-|-|
|Trac ID|trac#1063|
|Type|defect|
|Created|2005-05-28 07:16:48Z|

<details><summary>Searchable metadata</summary>

```
trac-id__1063 1063
type__defect defect
reporter__bruce bruce
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1117264608000000 1117264608000000
changetime__1200281566000000 1200281566000000
version__ 
owner__mesozoic mesozoic
cc__exarkun cc__mesozoic cc__bruce
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="bruce's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|bruce commented|
|-|-|

```
#!html
<pre>
Self-evident:

Index: twisted/internet/defer.py
===================================================================
--- twisted/internet/defer.py   (revision 13543)
+++ twisted/internet/defer.py   (working copy)
@@ -164,8 +164,8 @@
     that come from outside packages that are not under our control, we use
     threads (see for example L{twisted.enterprise.adbapi}).

-    For more information about Deferreds, see doc/howto/defer.html or
-    U{http://www.twistedmatrix.com/documents/howto/defer}
+    For more information about Deferreds, see doc/core/howto/defer.xhtml or
+    U{http://twistedmatrix.com/projects/core/documentation/howto/defer.html}
     """
     called = 0
     default = 0
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun commented|
|-|-|

```
#!html
<pre>
We need stable URLs we can link to from the docs.
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun set status to `closed`|
|-|-|

Fixed by r14294, [#10751](https://github.com/twisted/twisted/issues/10751)
